docs/lcdproc-config.5.in

```diff
@@ -423,6 +423,12 @@ Screen example (depending on screen size):
 .RS 4
 Show the screen [default: false; legal: true, false]
 .RE
+.PP
+\fIIgnore=\fR
+.RS 4
+Directs the Disk screen to ignore the specified mount point. [defaults: /boot/efi, /dev]
+You may have up to 10 Ignore lines under the Disk section.
```

Comment:
defaults is not applicable here. (It would only be, if there were hard coded defaults in the code, not for the default configuration file.) I can clean this up during merging.